Dockerfile

```diff
@@ -1,6 +1,7 @@
 # Build Postgres
 #
-FROM zimg/rust:1.56 AS pg-build
+#FROM zimg/rust:1.56 AS pg-build
+FROM zenithdb/build:buster-20220309 AS pg-build
```

Comment:
I've removed the corresponding entry in README, so now would be good to have a comment somewhere about this image and where does it come from.

Unless we plan to move onto zimg soon, then it does not make much sense.